tests/test_jax_nonbonded.py

```diff
@@ -334,7 +337,38 @@ def u_b(x, box, params):
 
         return jnp.sum(vdw + es)
 
-    np.testing.assert_almost_equal(u_a(conf, box, params), u_b(conf, box, params))
+    np.testing.assert_allclose(u_a(conf, box, params), u_b(conf, box, params))
+
+
+def test_lj_basis():
+    """Randomized test that LJ(sig, eps) computed via basis expansion matches reference"""
+
+    np.random.seed(2023)
+
+    n_env = 10_000
+    r_i = np.linspace(1.9, 8.0, n_env)
```

Comment:
nit: comment on why these values were chosen?

Reply:
Willfix.

(This range was inherited from an initial plot using combined_sigma = fixed_value_near_1 + variable_value_near_1`. The lower end of the range was chosen to be just under 2 to avoid too much clashing, and the upper end of the range was chosen arbitrary and large.)